### ISSUE TITLE
Token Similarity in the paper

### ISSUE THREAD

**Issue by pykao** [rccchoudhury/rlt#5]
Great work! In the paper, you are calculate the Patch similarity between two consecutive patches P1, and P2. 

In the formula 1, is x, y are all location within the patch? These two patches are considered similar patches if all x, y are smaller than threshold tau?

BR, 
PYK

**Comment by rccchoudhury** [comment]
No, (X, Y) are the coordinates of the patch within the frame. Each frame is divided into 16x16 patches, with each patch assigned an x, y coordinate corresponding to its center. We determine similarity by computing the L1 difference between the pixels in patch 1 and patch 2 and computing their average.
